source/main/index.js

```diff
@@ -175,13 +175,13 @@ const onAppReady = async () => {
     await safeExit();
   });
 
-  buildAppMenus(mainWindow, cardanoNode, locale);
+  buildAppMenus(mainWindow, cardanoNode, locale, false);
```

Comment:
@tomothespian filling in params directly is a bad practice. Please replace `false` with something like:
```
 buildAppMenus(mainWindow, cardanoNode, locale, isAppUpdateAvailable  = false);
```